mantidproject/mantid
# Workbench linux install

The purpose of this work is to add the workbench to the nightly installer for rhel7, but changes to the ubuntu installer were also done. Since `buildscript` already enables the workbench on PRs and master, that script does not need to be altered.

Small cleanup in the other linux launch scripts happened as well. Notably, `launch_mantidplot.sh` is now installed as `mantidplot` so there is one less symbolic link created.

**To test:**

Run the rhel7 installer (build and run for ubuntu as well) somewhere and see that the following all "work" without errors or warnings.
* `/opt/mantidunstable/bin/mantidplot`
* `/opt/mantidunstable/bin/MantidPlot`
* `/opt/mantidunstable/bin/mantidpython`
* `/opt/mantidunstable/bin/mantidworkbench`

Bonus items for RHEL7
* `/usr/bin/mantidplotunstable`
* `/usr/bin/mantidpythonunstable`
* `/usr/bin/mantidworkbenchunstable`


**Tested supported platforms:**
* RHEL7 *github username*
* Ubuntu 16.04 @martyngigg
* Ubuntu 18.04 @AndreiSavici 
* Windows (workbench not in installer, but rest should still work) @martyngigg 
* MacOS (workbench not in installer, but rest should still work) @quantumsteve 

*There is no associated issue.*

*This should not be in the release notes yet* because it is a step in the direction of having something for users to try and comment on.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
